gsoc/2018.md

```diff
@@ -81,6 +81,37 @@ Scalafix is a fast moving project, so you will be working in a codebase with mul
 
 Supervised by [@olafurpg](https://github.com/olafurpg)
 
+### Fox, a documentation engine for Scala
+
+Documentation may be consumed and written in many ways. Library authors can
+write tutorials, in-depth guides, or Scala docs; and users can consume them in
+PDF files, online websites or ebook readers. There are many hidden requirements
+in writing good documentation and, given how difficult it is to get it right,
+it is a task that many developers eventually ignore or, at best, procrastinate
+at.
+
+This project calls for the creation of a flexible documentation engine that
```

Comment:
I think it's good to mention that there already exist a lot of fantastic doc tools like https://www.gatsbyjs.org/, http://www.mkdocs.org/ and https://gohugo.io/ . However, none of them integrate nicely with the Scala toolchain like sbt/tut.